cluser_validation_checklist.md

```diff
@@ -0,0 +1,87 @@
+# Cluster Validation Checklist
+## Check general cluster status
+
+ - [ ] Review installation log
+   Check for any glaring issues, cluster admin credentials can be pulled from here as well.
+   Install logs can be found on the install node, inside of the install directory
+   ```shell
+   $ cat <install_dir>/.openshift_install.log
+   ```
+
+ - [ ] Verify that image pull source is what is expected
+   Check that the source of your openshift images is as expected.
+   As we are not in a disconnected environment quay.io and the internal cluster registry.
+   ```shell
+   $ oc adm node-logs <node_name> -u crio
+   ```
+
+ - [ ] Verify that cluster version matches what is expected
+   We'll be running the latest generally available version of OpenShift.
```

Comment:
We won't always be running the latest GA when we use this checklist.  Please reference what the person running the checklist woudl find the intended version.  (which file in this repo?)